tests/performance/CMakeLists.txt

```diff
@@ -5,9 +5,16 @@
 
 set(tests
     blaze_benchmarks
+    dist_cannon_4_loc
+    dist_cannon_9_loc
+    dist_cannon_16_loc
     simple_loop
    )
 
+set(dist_cannon_4_loc_PARAMETERS LOCALITIES 4)
+set(dist_cannon_9_loc_PARAMETERS LOCALITIES 9)
+set(dist_cannon_16_loc_PARAMETERS LOCALITIES 16)
```

Comment:
Not sure if our testing infrastructure can handle this. But we'll see.

Reply:
I think performance tests are not added for typical runs, we don't put `  add_phylanx_unit_test("performance" ${test} ${${test}_PARAMETERS})` in their cmake, so we can run them only when we want.

Reply:
Fair enough!